container-core/src/main/java/com/yahoo/container/core/documentapi/DocumentAccessProvider.java

```diff
@@ -27,18 +27,18 @@
  *
  * @author jonmv
  */
-public class MessageBusDocumentAccessProvider extends AbstractComponent implements Provider<DocumentAccess> {
+public class DocumentAccessProvider extends AbstractComponent implements Provider<DocumentAccessProvider.LazyWrapper> {
```

Comment:
This lets us use this in internal classes as well, without fear that the user may have configured their own `DocumentAccess` component.